### PR TITLE
feat: Add menu item for copyright information #1934

### LINKED ISSUE #1934: Add menu item for copyright information
**Actual Behaviour**

There is no menu item for displaying copyright information that is in event modal.

**Expected Behaviour**

There should be an overflow menu on the home screen which displays copyright information when clicked.

**Would you like to work on the issue?**
Yes

To be implemented after #1680

### PR DESCRIPTION
**Pull request by AkshayCHD**
Fixes #1934 

Changes: An overflow menu option added in home screen that displays the copyright information in a dialog.

Screenshots for the change: 
screenshot_1506949956
screenshot_1506949958

### CODE REVIEW

**Comment by AkshayCHD** [comment]
@iamareebjamal Sir if you could review it now, and sorry for sending the previous one with so many errors, this being my first contribution i was unaware of those minor changes.

**Comment by iamareebjamal** [comment]
No problem @AkshayCHD 

However it is generally preferred to only keep 1 PR of an issue, and instead of closing it, you should update it

File: android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java [hunk 1]
```diff
@@ -134,6 +139,8 @@
     private Dialog dialogNetworkNotification;
     private FragmentManager fragmentManager;
 
+
+
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: Please remove these extra lines

File: android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java [hunk 2]
```diff
@@ -328,6 +335,37 @@ private void setNavHeader(Event event) {
         }
     }
 
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.menu_extras, menu);
+        return true;
+
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: Please remove this extra line

File: android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java [hunk 3]
```diff
@@ -328,6 +335,37 @@ private void setNavHeader(Event event) {
...
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        displayCopyrightInformation();
+        return true;
+    }
+    private void displayCopyrightInformation(){
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: Please add a line above it and a space after `)`

**Comment by AkshayCHD** [comment]
@iamareebjamal Sorry sir didn't know

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
Great! Just a few changes will make it perfect :smile:

**Comment by AkshayCHD** [comment]
@iamareebjamal sir finally i think all changes are complete

**Review by iamareebjamal — APPROVED** [review]

File: android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java [hunk 4]
```diff
@@ -327,6 +328,36 @@ private void setNavHeader(Event event) {
...
     }
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.menu_extras, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        displayCopyrightInformation();
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: Please add a switch here with selecting id of copyright only

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java (hunk 4 above)
Comment (AkshayCHD) [reply android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: @iamareebjamal done sir please have a look

**Comment by AkshayCHD** [comment]
@iamareebjamal sir please if you could review now.

**Review by scandindian — APPROVED** [review]

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**